modules/library/render/src/test/java/org/geotools/renderer/lite/gridcoverage2d/GridCoverageRenderingOutOfViewTest.java

```diff
@@ -0,0 +1,101 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ * 
+ *    (C) 2016, Jumbletree Ltd

```

Comment:
@matchami there is a problem with this copyright header. To contribute code to the GeoTools project, copyright must be assigned to OSGeo. Source code files must use the following header:
http://docs.geotools.org/latest/developer/introduction.html#source-license

As you are adding a new file, please submit a code contribution agreement. If this work is owned by Jumbletree Ltd, then use the corporate agreement signed by an authorised person:
http://docs.geotools.org/latest/developer/procedures/contribution_license.html

Please also copy @jodygarnett (OSGeo project officer) when you submit your agreement.

Sorry for not spotting this earlier.

I also urge you to squash these commits into a single commit for clarity. You can do this with ```git cherry-pick -n COMMIT_ID ...``` to prepare your changes and then combine them into a single commit.

I also see that you are working on master in your fork. One of the big changes in Git (from svn) is that branches are super-easy to create and use, and better than working on master. For example, it is easy to create a branch ```geot-5421-master```, make your changes as a single commit, push to your GitHub fork, and make a pull request. This allows you to work on your next issue while your pull request is outstanding.